Exploration_Baseball_CorrelationsMocapRadar.R

```diff
@@ -238,37 +107,6 @@ allDat %>%
   geom_smooth(method=lm) + 
   stat_cor(method = "pearson")
 
-allDat %>%
-  filter(footNegWork > -20000)%>%
-  ggplot(aes(x= footNegWork,y=Velocity)) +
-  geom_point(alpha=0.5) +
-  labs(x= "Lead Distal Foot Neg Work", y="Throw Velocity (mph)")+
-  geom_smooth(method=lm) + 
-  stat_cor(method = "pearson")
-
-allDat %>%
-  filter(leadFootEnergy < 50)%>%
-  ggplot(aes(x= leadFootEnergy,y=Velocity)) +
-  geom_point(alpha=0.5) +
-  labs(x= "Lead Foot Energy", y="Throw Velocity (mph)")+
-  geom_smooth(method=lm) + 
-  stat_cor(method = "pearson")
-
-allDat %>%
-  filter(leadShankEnergy < 100)%>%
-  ggplot(aes(x= leadShankEnergy,y=Velocity)) +
-  geom_point(alpha=0.5) +
-  labs(x= "Lead Shank Energy", y="Throw Velocity (mph)")+
-  geom_smooth(method=lm) + 
-  stat_cor(method = "pearson")
-
-allDat %>%
-  #filter(leadThighEnergy < 50)%>%
-  ggplot(aes(x= leadThighEnergy,y=Velocity)) +
-  geom_point(alpha=0.5) +
-  labs(x= "Lead Thigh Energy", y="Throw Velocity (mph)")+
-  geom_smooth(method=lm) + 
-  stat_cor(method = "pearson")
 
 allDat %>%
```

Comment:
Getting error" Error: Problem with `filter()` input `..1`.
i Input `..1` is `rearFootEnergy < 20`.
x object 'rearFootEnergy' not found "

Reply:
Getting the same error for Shank and Thigh rear energy